tests/vmi_multus_test.go

```diff
@@ -321,16 +321,16 @@ var _ = Describe("Multus", func() {
 				}, 15)
 				Expect(err).ToNot(HaveOccurred())
 
-				By("Verifying the desired custom MAC is not configured inside the pod namespace.")
+				By("Verifying the interface is not configured inside the bridge forward database.")
 				vmiPod := tests.GetRunningPodByVirtualMachineInstance(vmiOne, tests.NamespaceTestDefault)
 				out, err := tests.ExecuteCommandOnPod(
 					virtClient,
 					vmiPod,
 					"compute",
-					[]string{"sh", "-c", "ip a"},
+					[]string{"sh", "-c", fmt.Sprintf("bridge fdb | grep eth0 | grep %s | wc -l", customMacAddress)},
```

Comment:
Would I now still see the custom mac address on the interface? Should you test for that too that it is kept on the interface?

Reply:
yeah sure I can do it I will leave the original test and also add the `bridge fdb` check